From 22a2f81f0d5521d62955f4acf3ccb61e47c69eda Mon Sep 17 00:00:00 2001
From: Ranbir Singh <Ranbir.Singh3@Dell.com>
Date: Tue, 24 Jan 2023 14:46:31 +0530
Subject: [PATCH 1/3] MdeModulePkg/.../GraphicsConsoleDxe: Fix UNUSED_VALUE
 Coverity issue

Full pathname - MdeModulePkg/Universal/Console/GraphicsConsoleDxe

The return value stored in "Status" after call to the function
UgaDraw->Blt is not made of any use thereafter, it remains as UNUSED
and later it gets overriden.

Assuming this is deliberate, the storage in "Status" can be removed.

Cc: Zhichao Gao <zhichao.gao@intel.com>
Cc: Ray Ni <ray.ni@intel.com>
REF: https://bugzilla.tianocore.org/show_bug.cgi?id=4291
Signed-off-by: Ranbir Singh <Ranbir.Singh3@Dell.com>
---
 .../GraphicsConsoleDxe/GraphicsConsole.c      | 24 +++++++++----------
 1 file changed, 12 insertions(+), 12 deletions(-)

diff --git a/MdeModulePkg/Universal/Console/GraphicsConsoleDxe/GraphicsConsole.c b/MdeModulePkg/Universal/Console/GraphicsConsoleDxe/GraphicsConsole.c
index b895dafedeaa..dea310e237f9 100644
--- a/MdeModulePkg/Universal/Console/GraphicsConsoleDxe/GraphicsConsole.c
+++ b/MdeModulePkg/Universal/Console/GraphicsConsoleDxe/GraphicsConsole.c
@@ -1454,18 +1454,18 @@ GraphicsConsoleConOutSetMode (
       //
       // The current graphics mode is correct, so simply clear the entire display
       //
-      Status = UgaDraw->Blt (
-                          UgaDraw,
-                          (EFI_UGA_PIXEL *)(UINTN)&mGraphicsEfiColors[0],
-                          EfiUgaVideoFill,
-                          0,
-                          0,
-                          0,
-                          0,
-                          ModeData->GopWidth,
-                          ModeData->GopHeight,
-                          0
-                          );
+      UgaDraw->Blt (
+                 UgaDraw,
+                 (EFI_UGA_PIXEL *)(UINTN)&mGraphicsEfiColors[0],
+                 EfiUgaVideoFill,
+                 0,
+                 0,
+                 0,
+                 0,
+                 ModeData->GopWidth,
+                 ModeData->GopHeight,
+                 0
+                 );
     }
   }
 
-- 
2.36.1.windows.1

